js/components/Tab.jsx

```diff
@@ -0,0 +1,39 @@
+var React = require("react");
+
+var Tab = function(props) {
```

Comment:
Is that a temporary element? Looks a bit like the "style" doesn't really have a place in a generic component if it was truly a generic "tab".